src/core/java/com/enderio/core/client/item/ItemTooltip.java

```diff
@@ -0,0 +1,30 @@
+package com.enderio.core.client.item;
+
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.fluids.FluidStack;
+
+
+import java.util.List;
+
+public class ItemTooltip {
```

Comment:
We already have a lot of tooltip functionality as a part of the IAdvancedTooltipProvider interface, I'd recommend a little read over how this works, most of the tools use it.